drivers/volume/portworx/portworx.go

```diff
@@ -2996,6 +3023,139 @@ func configurePxReleaseManifestEnvVars(origEnvVarList []corev1.EnvVar, specGenUR
 	return newEnvVarList, nil
 }
 
+// isPxCustomRegistryConfigMapExist return true or false based wether this custom registry configmap exists or not
+func isPxCustomRegistryConfigMapExist(pxNamespace string) bool {
+	// Get custom registry configmap configmap
+	_, err := core.Instance().GetConfigMap(pxCustomRegistryConfigmapName, pxNamespace)
+	if k8serrors.IsNotFound(err) {
+		return false
+	}
+	return true
+}
+
+// isPxVersionsConfigMapExist( return true or false based on wether px-versions cm exists or not
+func isPxVersionsConfigMapExist(pxNamespace string) bool {
```

Comment:
why two methods to check two different configMap ? we can have single method and passing the configMap name as parameter ..

Reply:
Good catch! Added the second one last minute when I found a bug and didn't realize I am looking for same thing in both functions!